Client/Frontend/Browser/Playlist/VideoPlayer/UI/VideoPlayer.swift

```diff
@@ -545,23 +546,14 @@ class VideoView: UIView, VideoTrackerBarDelegate {
   }
 
   private func toggleOverlays(showOverlay: Bool, except: [UIView] = [], display: [UIView] = []) {
-    var except = except
-    var display = display
-
-    if delegate?.isVideoTracksAvailable == true {
-      if showOverlay {
-        except.append(particleView)
-      }
-    } else {
-      // If the overlay is showing, hide the particle view.. else show it..
-      except.append(particleView)
-      display.append(particleView)
-    }
-
     UIView.animate(
       withDuration: 1.0, delay: 0.0, usingSpringWithDamping: 1.0, initialSpringVelocity: 1.0, options: [.curveEaseInOut, .allowUserInteraction],
       animations: {
         self.subviews.forEach({
+          if $0 == self.particleView {
```

Comment:
Should we not use reference equality `===` here?

Reply:
`NSObject` subclass equality is based on pointers by default so they would be the same

Reply:
in the repo we do compare by === reference for `Tab` objects which subclasses `NSObject` as well, was thinking this is a good convention in general, could cause undesired behavior if we conform to a custom Equatable comparison method or overriding `isEqual`, unlikely for this view of course